website/docs/r/analysis_services_server.html.markdown

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
could we add this new resource [to the sidebar](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L389-L398)?